src/js/components/viewer/models/audio-media-object.jsx

```diff
@@ -149,23 +164,24 @@ var AudioMediaObject = React.createClass({
 
                 this.load(streamUrl);
 
-                // APEP TODO resolve audio volume bug then we can turn this back on
-                this.volume(volume);
+                // APEP set the volume as 0 and allow the audio tween to ramp up to the authored volume
+                this.volume(0);
 
                 self.unlockCuePointsForMediaObject(mediaObject);
 
-                this.play();
-
-                //APEP tween volume TODO Turn back on with time to diagnose
+                //APEP tween volume
                 var _ops = self.props.data.mediaObject._ops;
-                // self.state.playingAudioTween = self.tweenAudio(0, volume, _ops.transitionDuration, this);
-                // self.state.playingAudioTween.start();
+                var firstPlayAudioTween = self.tweenAudio(0, volume, _ops.transitionDuration, this);
+                self.setState({firstPlayAudioTween: firstPlayAudioTween});
```

Comment:
I have tested this against the scene AP_BEACH_FRONT_SOUNDS, all sounds were playing full volume.
I have reset cache, node modules and dist, that did not change the result might be worth investigating